packages/core/src/adapters.ts

```diff
@@ -204,6 +204,10 @@ export interface AdapterAccount extends Account {
  * A session holds information about a user's current signin state.
  */
 export interface AdapterSession {
+  /**
+   * Session id. Only used when using `"database"` `AuthConfig.strategy` option.
+   */
+  id: string;
```

Comment:
If this is 

```suggestion
  id?: string;
```

it should be backwards compatible